### ISSUE TITLE
Accordion focus color could be improved for accessibility

### ISSUE THREAD

**Issue by starryeyez024** [patternfly/patternfly-elements#444]
I think the light-gray is very difficult to see
![image](https://user-images.githubusercontent.com/456863/60209763-5180fe00-9829-11e9-8a7b-a3bb68e57743.png)

The contrast ratio is 1.7:1 which fails AA accessibility rating.

suggesting ui-link blue for underline
![image](https://user-images.githubusercontent.com/456863/60209790-59d93900-9829-11e9-83ae-243ccddc12af.png)
